pkg/cli/admin/release/mirror.go

```diff
@@ -251,7 +262,20 @@ func (o *MirrorOptions) Complete(cmd *cobra.Command, f kcmdutil.Factory, args []
 		client := coreClient.ConfigMaps(configmap.NamespaceLabelConfigMap)
 		return client, nil
 	}
-	o.PrintImageContentInstructions = true
+	if o.PrintImageSourceInstructions == "" {
+		fmt.Fprintf(o.Out, "WARNING: ImageContentSources and ImageDigestMirrorSet in the default printed out mirror instructions are deprecated and will be removed in a future release. Use --print-mirror-instructions=idms to allow the printing of instructions for ImageDigestSources and ImageDigestMirrorSet.\n")
```

Comment:
We are already printing a warning in Line#272. Instead printing warning twice, we can modify the one below and remove this one.